src/modules/addToDo.js

```diff
@@ -24,6 +26,29 @@ const addToDoList = (todoValue) => {
       deleteItem(currentListitem);
     });
   });
+
+  checkBox.forEach((check) => {
+    const currentListItem = check.closest('.list-item');
+    checkCompletedTask(check, currentListItem);
+    check.addEventListener('change', (event) => {
+      completedTask(currentListItem, event);
+    });
+  });
 };
 
+// toDoList.addEventListener('click', (e) => {
+//   if (e.target.classList.contains('checkbox')) {
+//     const currenTrashIcon = document.querySelectorAll('.bi-trash');
+//     const currentDotIcon = document.querySelectorAll('.bi-three-dots-vertical');
+//     currentDotIcon.forEach((icon) => {
+//       icon.classList.toggle('hide');
+//     });
+//     currenTrashIcon.forEach((trash) => {
+//       trash.classList.toggle('active');
+//     });
+//     const currentCheckBox = e.target.closest('.list-item');
+//     currentCheckBox.classList.toggle('cancelList');
+//   }
+// });
+
```

Comment:
- [ ] [REQUIRED] Please, remove commented code because it makes the code base not clean.

Reply:
@OmarRamoun Thanks